source/seedsource/report.py

```diff
@@ -191,6 +219,145 @@ def get_pdf_data(self) -> BytesIO:
+
+        t = ctx['today']
+        date = t.strftime('%m/%d/%Y')
+        attribution = '{} on {}'.format(ATTRIBUTION_TEXT, date)
+
+        # Set map slide placeholders
+        mapslide = prs.slides[0]
+        placeholder = mapslide.placeholders[MAP_IDX]
+        placeholder.insert_picture(ctx['image_data'])
+        placeholder = mapslide.placeholders[SCALE_IDX]
+        placeholder.text = ctx['scale']
+        placeholder = mapslide.placeholders[WEST_IDX]
+        placeholder.text = degree_sign(ctx['west'])
+        placeholder = mapslide.placeholders[NORTH_IDX]
+        placeholder.text = degree_sign(ctx['north'])
+        placeholder = mapslide.placeholders[EAST_IDX]
+        placeholder.text = degree_sign(ctx['east'])
+        placeholder = mapslide.placeholders[SOUTH_IDX]
+        placeholder.text = degree_sign(ctx['south'])
+        placeholder = mapslide.placeholders[ATTRIBUTION_IDX]
+        placeholder.text = attribution
+
+        # Gather all text for mapslide.notes_slide with all the stat info used below and in statslide
+        objective = ctx['objective']
+        location_label = ctx['location_label'] + ':'
+        point = ctx['point']
+        point_label = 'Lat: {y}{degree_sign}, Lon: {x}{degree_sign}'.format(x=point['x'], y=point['y'],
+                                                                            degree_sign=DEGREE_SIGN)
+        elev_label = '{} ft'.format(ctx['elevation'])
+        seedlot_year = ctx['seedlot_year']
+        site_year = ctx['site_year']
+        tmethod = ctx['method']
+        center = ctx['center']
+
+        # Determine method text
+        if tmethod == 'seedzone':
+            if center == 'zone':
+                method_text = 'Transfer limits and climatic center based on seed zone'
+            else:
+                method_text = 'Transfer limits based on seed zone, climatic center based on the selected location'
+        else:
+            method_text = 'Custom transfer limits, climatic center based on the selected location'
+
+        lines = [
+            ('Objective: ', objective, True),               # (bold_text, norm_text, add_newline)
+            (location_label + ' ', point_label, False),
+            ('Elevation: ', elev_label, True),
+            ('Climate Scenarios', '', False),
+            ('', 'Seedlot climate: ' + seedlot_year, False),
+            ('', 'Planting site climate: ' + site_year, True),
+            ('Transfer limit method: ', method_text, True)
+        ]
+
+        if tmethod == 'seedzone':
+            species_text = 'Species: '
+            species_label = ctx['species']
+            sz_text = 'Seed zone:'
+            band_str = ", {}' - {}'".format(ctx['band'][0], ctx['band'][1]) if ctx['band'] else ''
+            sz_label = '{}{}'.format(ctx['zone'], band_str)
+            lines += [
+                (species_text, species_label, False),
+                (sz_text, sz_label, False)
+            ]
+
```

Comment:
Add lines for listing variables in the mapslide notes. Probably want to handle this separate in the `add_text_frame` call since we're bolding the first row rather than the first column. Format could be:

**Variable | Center | Transfer Limit (+/-)**
---------------------------------------------    (however long this needs to be, based on variable name
variable 1 | center 1 | t_limit 1
etc...